data/intro missions.txt

```diff
@@ -451,3 +451,143 @@ conversation "end of intro missions"
+			choice
+				`	"Why don't you come with me and see them for yourself?  There are a lot of retired spacers on <planet>."`
+				`	"It has been good catching up with you, James."`
+					goto end
+			`	James agrees. When you ask if he needs to collect any of his belongings before the trip he declines, but buys a simple traveling hygiene kit from one of the spaceport shops.`
+				accept
+			label end
+			`	You finish your conversation and go on your way`
+				decline
+	on decline
+		log `Paid a visit to James on <origin> and told him about the Hai.`
+	on complete
+		conversation
+			`James is impressed by the friendly aliens and humans living side by side on <planet>. While you are introducing him to one of the Hai, an old spacer recognizes James and offers to make further introductions around the community on <planet>. You make your goodbyes once again, and prepare to continue on your way.`
+		log `Paid a visit to James on <origin> to tell him about the Hai. Brought him to <planet> afterwards.`
+
+
+
+mission "Intro: Follow Up - After Main Plot at Allhome"
+	invisible
+	landing
+	source "Allhome"
+	to offer
+		has "Intro: Follow Up - Hai Before Main Plot: done"
+		has "main plot completed"
+	to fail
+		has "Intro: Follow Up - Hai After Main Plot"
+	on offer
+		conversation
+			`It has been some time since you brought James to <origin>. Do you want to check in and see how he is doing?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			`	You meet up with James at the spaceport, where he is sharing a drink with some other merchant captains. The group disperses to let the two of you catch up on old times, so you fill James in on some of the details of the recent war and the role you ended up playing in it. You talk a bit about the Pug and what you might do next, now that the war is behind you. Soon it is time for you to go and the two of you part ways again.`
+				complete
+	on complete
+		log `Met with James on <origin>, and filled him in on recent events with the war and the Pug.`
+
+
+
+mission "Intro: Follow Up - After Main Plot"
+	invisible
+	landing
+	source "Hestia"
+	to offer
+		or
+			has "Intro [3 Freighter]: done"
+			has "Intro [3 Transport]: done"
+			has "Intro [3 Interceptor]: done"
+		has "main plot completed"
+	to fail
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`As you're coming in for a landing, you remember that James left you an open invitation to look him up. Do you want to check in and see how he is doing?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			`	James is happy to come out to the spaceport and meet with you. You spend some time talking about recent events, and fill him in on some of the details of the recent war and the role you ended up playing in it. You talk a bit about the Pug and what you might do next, now that the war is behind you.`
+				complete
```

Comment:
`complete` isn't an endpoint condition. This should be `decline`, and the line below should be changed to `on decline`.

Reply:
done